ethToElrond/v2/errors.go

```diff
@@ -2,11 +2,16 @@ package v2
 
 import "errors"
 
-var (
-	errNilBatch            = errors.New("nil batch")
-	errInvalidDepositNonce = errors.New("invalid deposit nonce")
-	errNilLogger           = errors.New("nil logger")
-	errNilElrondClient     = errors.New("nil Elrond client")
-	errNilEthereumClient   = errors.New("nil Ethereum client")
-	errNilTopologyProvider = errors.New("nil topology provider")
-)
+// TODO add comments
```

Comment:
Implement TODOs :D
Also, why do we need them exported?

Reply:
those are defined in /v2/errors and we use them in /v2/elrondToEth/steps